plugin.cpp

```diff
@@ -809,7 +885,7 @@ void plugin_reconfigure(PLUGIN_HANDLE *handle, const string& newConfig)
 				std::regex re(asset_name); // Check if regex is valid
 				std::regex re2(datapoint); // Check if regex is valid
 				Logger::getLogger()->info("Parse asset filter config, Adding to assetFilterConfig map: {%s, %d, %s}", asset_name.c_str(), actn, new_asset_name.c_str());
-				(*newInfo->assetFilterConfig)[asset_name] = {actn, new_asset_name, dpmap, datapoint, dpType, splitAssets};
+				(*newInfo->assetFilterConfig)[asset_name] = {actn, new_asset_name, dpmap, datapoint, dpType, splitAssets, selection};
```

Comment:
Should datapoint in selection vector support regex ?

Reply:
There is no requirement to have the datapoints expressed as a regex currently. It would have a considerable impact on performance to do this, so I am tended to say no currently.